### ISSUE TITLE
org.jsoup.nodes.Element.wholeText() not correctly translating <br> ?

### ISSUE THREAD

**Issue by JupiterRider** [jhy/jsoup#1437]
`		String s = Jsoup.parse("Hello<br>World").wholeText();`
		`System.out.println(s);`

prints `HelloWorld`. 

But shouldn't it be
`Hello`
`World` ?

There is no line separator.

![firefox](https://user-images.githubusercontent.com/60042618/97023682-24944680-1556-11eb-9210-c8a93439d9e1.PNG)

![ide](https://user-images.githubusercontent.com/60042618/97023712-2eb64500-1556-11eb-84eb-d98cea55b974.PNG)

**Comment by suarez12138** [comment]
Maybe it can be solved by adding detection of special tags like `<br>` and convert them manually in `wholeText()`. 

```JAVA
public void head(Node node, int depth) {
                if (node instanceof TextNode) {
                    TextNode textNode = (TextNode) node;
                    accum.append(textNode.getWholeText());
                }else if (node.nodeName().equals("br")){
                    accum.append("\n");
                }
            }
```

But whether there is a need to do it? And deal with more special tags like `<hr>` or just `<br>` ?

**Comment by jhy** [comment]
Thanks this was fixed in #1636
